addons/context_actions/functions/fnc_repairVehicles.sqf

```diff
@@ -0,0 +1,22 @@
+/*
+ * Author: 3Mydlo3
+ * Repairs units in given selection.
+ *
+ * Arguments:
+ * N: Selected Objects <OBJECT>
```

Comment:
This is wrong, it's ARRAY of objects.
Just passing an object would make the select error out

Reply:
Whoops, missed this in all functions probably. Sorry, I'll fix this later today. Hope no other issues or such mistakes were made.